inc/audiences/namespace.php

```diff
@@ -96,6 +96,7 @@ function register_post_type() {
 function maybe_grant_caps( $allcaps ) {
 	$cap_map = [
 		'edit_audiences' => 'edit_pages',
+		'read_audiences' => 'read',
```

Comment:
I'm not sure if this is necessary or if it's implicit. However, as the post type is not public, perhaps it actually is necessary.

Reply:
Best way to find out is to try removing it and see if everything still works